README.md

```diff
@@ -1 +1,33 @@
-# k8s
\ No newline at end of file
+# Production staging example
+To run the demo checkot the prod_stage branch:
+```
+git checkout prod_stage
+cd prod_stage
+```
+Make sure you have minikube up and running:
+```
+minikube status
+```
+Open a new terminal window and start minikube tunnel (it's a blocking operation)
+```
+minukube tunnel
+```
+Build the needed images
+```
+cd web_app
+docker build . -t web-app
+cd ..
+```
+Export the minikube docker-env variables
+```
+eval $(minikube docker-env)
+```
```

Comment:
`eval` has to be executed before `docker build . -t web-app`

So just put the `eval` step, before the docker build!
thanks